### ISSUE TITLE
[Issues] Important Issues

### ISSUE THREAD

**Issue by Eplen** [GC-spigot/Tokens#88]
**Describe the bug**
1. Enchantments
Enchantments doesnt work, like Explosion or JackHammer, or somethig like those with particles and doing stuf, you know what i mean. Even when i have Tokens Region set-up, it doesnt work. The full error message is down there. Idk if its caused by this, but it just doesnt work. 

2. Kit
I created an kit, where is pickaxe with enchantments, but it just gives the lore from tokenskitaddons to the pickaxe, but not the real enchantments. 

But when i add another level, it just like refresh and the enchantments is working and its on 4th level, video is down there about it.

There is my configuration: 
![image](https://user-images.githubusercontent.com/73800564/118413949-2ebc9480-b6a2-11eb-8366-c2d43b005a58.png)
My configuration in pastebin:
https://pastebin.com/WrnFyDU4

**Screenshots / Video**

All the time when someone is mining with explosive, jackhammer or enchantments like that..
Enchantments that do something, like explode or something, hope you understand.

Only this
1. Enchantments
![image](https://user-images.githubusercontent.com/73800564/118412093-5f97cc00-b698-11eb-9b06-3c23ac459325.png)

This is the error in LOGS from it - https://pastebin.com/x9EfFay5

2.  Kit
YouTube video that i made:
https://www.youtube.com/watch?v=SYJcY3cgnwc

**Server Information (please complete the following information):**
 - Minecraft Version [Spigot 1.8.9]
- Plugin Version [2.9.4]

[EDIT]

Problem n.1 - I tried to solve it replacing the prisonmine plugin, with supported plugin called "Mines"
It works only on Explosive, it do break the blocks, but there is not an explosive particle. And enchantment JackHammer is not even working, the error of it is there - https://pastebin.com/SbCu7faK, when im mining it spams that.

Ill post it again, so you can see it better: https://pastebin.com/SbCu7faK

**Comment by GC-spigot** [comment]
Okay, this block breaking issue is because of a PrisonMines plugin. Are you using this? https://www.spigotmc.org/resources/prisonmines.4046/ if not please link your plugin and we can add support for it

**Comment by Eplen** [comment]
Yes, i am using this plugin, is something bad with it?

**Comment by Eplen** [comment]
So problem n.1 is bcs of PrisonMines, did i get it right?

**Comment by GC-spigot** [comment]
Yes, according to that plugin’s review it’s not supported at all by original developer

**Comment by Eplen** [comment]
Okey, i'll try to replace it  with one of plugins you are supporting.

And that problem n.2 will be solved?

**Comment by Eplen** [comment]
Problem n.1 - I tried to solve it replacing the prisonmine plugin, with supported plugin called "Mines"
It works only on Explosive, it do break the blocks, but there is not an explosive particle. And enchantment JackHammer is not even working, the error of it is there - https://pastebin.com/SbCu7faK, when im mining it spams that.

In console, it says this: [Wed, 19. May 2021 21:07:35 UTC ERROR] Could not pass event BlockBreakEvent to Tokens v2.10.3

Ill post it again, so you can see it better: https://pastebin.com/SbCu7faK


I edited it to the topic, so you can see it.

**Comment by Eplen** [comment]
Any updates?

**Comment by ghost** [comment]
This is because you're using essentialskits, it's the worst thing you could ever use.
Tested kits on my server with a pickaxe with about 20 enchants and they all worked fine.

**Comment by Eplen** [comment]
> This is because you're using essentialskits, it's the worst thing you could ever use.
> Tested kits on my server with a pickaxe with about 20 enchants and they all worked fine.

Im not using Essentials Kits, look closely on that screenshot!
![image](https://user-images.githubusercontent.com/73800564/120934049-136f0300-c6fd-11eb-99b0-7cb8fd006883.png)

**Comment by Firosee** [comment]
So today i went with this user and helped them resolve all the issues, The error relating to particles is an issue effecting some of the latest addons, the solution was to temporarily downgrade by one version until this can be resolved in an upcoming update.
